test/e2e/carousel.test.js

```diff
@@ -84,6 +110,38 @@ describe('Nuka Carousel', () => {
   describe('Button Navigation and Loading', () => {
     describe('transitionMode - scroll', () => {
       defaultNavigationAndLoading();
+
+      describe('when next button is pressed', () => {
+        it('should allow you to navigate entire slide deck when left aligned', async () => {
+          await expect(page).toClick('button', {
+            text: 'Toggle Partially Visible Slides'
+          });
+          await expect(page).toClick('button', { text: 'Left' });
+          await page.waitFor(600); // need to let slide transition complete
+
+          await nextThroughAllSlides();
+        });
+
+        it('should allow you to navigate entire slide deck when center aligned', async () => {
+          await expect(page).toClick('button', {
+            text: 'Toggle Partially Visible Slides'
+          });
+          await expect(page).toClick('button', { text: 'Center' });
+          await page.waitFor(600); // need to let slide transition complete
+
+          await nextThroughAllSlides();
+        });
```

Comment:
This test when the slides are `center` aligned specifically did _not_ work (the next button would disable too early) with the old logic, but now it works 😎